x-pack/plugins/fleet/server/services/agent_policy_watch.ts

```diff
@@ -75,69 +74,63 @@ export class PolicyWatcher {
   }
 
   public async watch(license: ILicense) {
-    let page = 1;
-    let response: {
-      items: AgentPolicy[];
-      total: number;
-      page: number;
-      perPage: number;
-    };
+    const agentPolicies: Array<Partial<AgentPolicy>> = [];
```

Comment:
you could define this array in a way that so that you don't need to type-cast further below. Like:

```typescript
const agentPolicies: Array<Pick<AgentPolicy, 'is_protected' | 'id'>> = [];
```